app/Models/User.php

```diff
@@ -1597,6 +1609,24 @@ public function playCount()
         return $this->memoized[__FUNCTION__];
     }
 
+    /**
+     * User's previous usernames
+     *
+     * @param bool $includeCurrent true if previous usernames matching the the current one should be included.
+     *
+     * @return \Illuminate\Database\Eloquent\Collection string
+     */
+    public function previousUsernames(bool $includeCurrent = false)
+    {
+        $query = $this->usernameChangeHistoryPublic;
+
+        if (!$includeCurrent) {
+            $query->where('username_last', '<>', $this->username);
```

Comment:
should be `$query = $query->where()` as it's now collection. And maybe rename the variable.